core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-content-id.service.ts

```diff
@@ -16,40 +16,47 @@
  * @fileoverview Service to get and set active content id in translation tab.
  */
 
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
 import { EventEmitter } from '@angular/core';
 
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-property.service.ts');
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-recorded-voiceovers.service.ts');
-
-angular.module('oppia').factory('TranslationTabActiveContentIdService', [
-  'StateRecordedVoiceoversService',
-  function(StateRecordedVoiceoversService) {
-    var activeContentId = null;
-    var activeDataFormat = null;
-    var _activeContentIdChangedEventEmitter = new EventEmitter();
-    return {
-      getActiveContentId: function() {
-        return activeContentId;
-      },
-      getActiveDataFormat: function() {
-        return activeDataFormat;
-      },
-      setActiveContent: function(contentId, dataFormat) {
-        var allContentIds = (
-          StateRecordedVoiceoversService.displayed.getAllContentId());
-        if (allContentIds.indexOf(contentId) === -1) {
-          throw new Error('Invalid active content id: ' + contentId);
-        }
-        activeContentId = contentId;
-        activeDataFormat = dataFormat;
-        _activeContentIdChangedEventEmitter.emit(dataFormat);
-      },
-      get onActiveContentIdChanged() {
-        return _activeContentIdChangedEventEmitter;
-      }
-    };
-  }]);
+import { StateRecordedVoiceoversService } from
+// eslint-disable-next-line max-len
+  'components/state-editor/state-editor-properties-services/state-recorded-voiceovers.service';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TranslationTabActiveContentIdService {
+  private activeContentId: string = null;
+  private activeDataFormat: string = null;
+  private _activeContentIdChangedEventEmitter = new EventEmitter();
```

Comment:
Some private properties have _ prefix and some don't, can you please check what's the expected pattern?

Reply:
Changed to private properties with _prefix and getters/setters without.